index.html

```diff
@@ -489,8 +489,10 @@ <h4 style="margin-top: 15px;">Follow Us</h4>
                 <ul class="links">
                     <li><a href="#" class="social"><i class="fa-brands fa-facebook-f"></i> Facebook</a></li>
                     <li><a href="#" class="social"><i class="fa-brands fa-instagram"></i> Instagram</a></li>
-                    <li><a href="#" class="social"><i class="fa-brands fa-linkedin-in"></i> Linkedin</a></li>
+                    <li><a href="#" class="social"><i class="fa-brands fa-linkedin-in"></i> LinkedIn</a></li>
+                    <li><a href="#" class="social"><i class="fa-brands fa-twitter"></i> Twitter</a></li> <!-- Added Twitter -->
```

Comment:
The Twitter icon has been updated. Please replace the existing icon with the latest version to ensure consistency.